Add CONTRIBUTING.md (#35)
factor10/intent: CONTRIBUTING.md

# Contributing to Intent

> The design of Intent and the structure of tests is still a moving target.

Author: ```suggestion
The design of Intent and the structure of tests are still moving targets.
```
